R/limits_to_agegroups.r

```diff
@@ -3,20 +3,37 @@
 #' Mostly used for plot labelling
 #' @param x age limits to transform
 #' @param limits lower age limits; if not given, will use all limits in `x`
-#' @return Age groups (limits separated by dashes)
+#' @param notation whether to use bracket notation, e.g. [0,4) or dash
+#'   notation, e.g. 0-4)
+#' @return Age groups as specified in `notation`
 #' @examples
 #' limits_to_agegroups(c(0, 5, 10))
 #' @export
-limits_to_agegroups <- function(x, limits = sort(unique(x))) {
+limits_to_agegroups <- function(x, limits = sort(unique(x)),
+                                notation = c("dashes", "brackets")) {
```

Comment:
Do we really need to keep both options? I'd recommend being more opinionated and stick with one to facilitate usage & maintenance.

Reply:
In the long run we may indeed want to remove this but for now I think we need this to support gradual deprecation.